examples/quarkus/testsuite/base/src/test/resources/application.properties

```diff
@@ -1,21 +1,21 @@
-%postgresql.quarkus.datasource.url=jdbc:postgresql://localhost:5432/test
-%postgresql.quarkus.datasource.driver=org.postgresql.Driver
```

Comment:
So with the new config, we don't need to define the driver anymore... but I saw in `SchemaClearer.java` that you used it.

I was able to have the h2 tests pass so I'm not sure if we need to keep it.

If you absolutely need it, we just need to change `quarkus.datasource.driver` to `quarkus.datasource.jdbc.driver`. I can do it, just let me know.